src/jit/lclvars.cpp

```diff
@@ -1769,6 +1789,7 @@ void Compiler::lvaPromoteLongVars()
         {
             if (lvaGetPromotionType(varDsc->lvParentLcl) != PROMOTION_TYPE_INDEPENDENT)
             {
+                JITDUMP("Not promoting long local V%02u: field of dependently-promoted struct\n", lclNum);
```

Comment:
Do you know if we hit this anywhere? I suspect that this will cause the same sorts of issues seen in #6778.